### PR TITLE
[DOC] Added link for tedana NeuroStars tag.

### LINKED ISSUE #593: Replace 'multi-echo' NeuroStars tag with 'tedana' on Support page
### Summary
@handwerkerd just opened a NeuroStars post about the `multi-echo` tag page not resolving correctly, which reminded me that there is a `tedana` tag that I think we should point users toward in our documentation. We could even ask folks to use `tedana` for tedana usage questions and `multi-echo` for general multi-echo questions.

### Additional Detail
None

### Next Steps
1. Update `docs/support.rst`.

### PR DESCRIPTION
**Pull request by aykhojandi**
Closes #593 .

Changes proposed in this pull request:

- Update support.rst file to include tedana tag link. 
- _Question:_ Should we also edit Line 7 to say "If you would like ... ``tedana`` tag." instead of "... ``multi-echo`` tag." ?(@jbteves and I were wondering this, given that we now want to point tedana-specific questions to the tedana tag).
-
-

### CODE REVIEW

**Comment by jbteves** [comment]
Hey @aykhojandi, thanks for your contribution! Two quick things:
1. Do you mind putting [DOC] at the start, rather than end, of your PR
2. Do you mind making this a Draft PR? Since we're unsure if it's actually ready for review.

**Comment by aykhojandi** [comment]
Got it, thanks. Made the necessary changes, let me know if something else
stands out as wrong. Best,
Arman

**Comment by tsalo** [comment]
> Question: Should we also edit Line 7 to say "If you would like ... tedana tag." instead of "... multi-echo tag." ?(@jbteves and I were wondering this, given that we now want to point tedana-specific questions to the tedana tag).

I would say something along the lines of "Please tag any posts about tedana usage with the `tedana` tag and any general questions about multi-echo fMRI with the `multi-echo` tag." Not that exactly, but basically distinguishing the two tags.

File: docs/support.rst [hunk 1]
```diff
@@ -4,9 +4,11 @@ Support and communication
 -------------------------
 All bugs, concerns and enhancement requests for this software can be submitted here: https://github.com/ME-ICA/tedana/issues.
 
-If you would like to ask a question about usage or tedana's outputs, please submit a question to `NeuroStars`_ with the ``multi-echo`` tag.
+If you would like to ask a question about tedana-specific usage or tedana's outputs, please submit a question to `NeuroStars`_ with the ``tedana`` tag.
```
Comment (tsalo) [inline docs/support.rst:7]:
```suggestion
If you would like to ask a question about tedana-specific usage or tedana's outputs, please submit a question to `NeuroStars`_ with the `tedana tag`_.
```

File: docs/support.rst [hunk 2]
```diff
@@ -4,9 +4,11 @@ Support and communication
...
 
-All previous tedana-related questions are available under the `multi-echo tag`_.
+If you would like to ask a general question about multi-echo fMRI, please submit it to `NeuroStars`_ with the ``multi-echo`` tag.
```
Comment (tsalo) [inline docs/support.rst:9]:
```suggestion
If you would like to ask a general question about multi-echo fMRI, please submit it to `NeuroStars`_ with the `multi-echo tag`_.
```

File: docs/support.rst [hunk 3]
```diff
@@ -4,11 +4,14 @@ Support and communication
...
+
+Note: All previous tedana-related questions are available under the `multi-echo tag`_.
 
 We will also attempt to archive certain common questions and associate answers in the Frequently Asked Questions (FAQ) page.
 
 .. _multi-echo tag: https://neurostars.org/tags/multi-echo
 .. _NeuroStars: https://neurostars.org
+.. _tedana: https://neurostars.org/tags/tedana
```
Comment (tsalo) [inline docs/support.rst:17]:
```suggestion
.. _tedana tag: https://neurostars.org/tags/tedana
```

**Review by tsalo — CHANGES_REQUESTED** [review]
A couple of minor adjustments to link to NeuroStars with each of the tags.

**Comment by tsalo** [comment]
@aykhojandi there's just one more change I'd like to see, but this PR is looking great! Do you want to mark it ready for review?

**Comment by aykhojandi** [comment]
> > Question: Should we also edit Line 7 to say "If you would like ... tedana tag." instead of "... multi-echo tag." ?(@jbteves and I were wondering this, given that we now want to point tedana-specific questions to the tedana tag).
> 
> I would say something along the lines of "Please tag any posts about tedana usage with the `tedana` tag and any general questions about multi-echo fMRI with the `multi-echo` tag." Not that exactly, but basically distinguishing the two tags.



> @aykhojandi there's just one more change I'd like to see, but this PR is looking great! Do you want to mark it ready for review?

Got it. I believe the last suggestion is now committed. I marked this ready for review. Let me know how else I can be of assistance.

**Comment by tsalo** [comment]
One related thing that would be really helpful is if you could also update the issue template (`.github/ISSUE_TEMPLATE/issue_template.md`), which asks users to use the "multi-echo" NeuroStars tag for usage-related questions. If you could change that to "tedana" that would be awesome.

**Review by tsalo — APPROVED** [review]
LGTM. Thanks for this!

**Review by emdupre — APPROVED** [review]
This LGTM :+1: Thank you, @aykhojandi !

**Comment by jbteves** [comment]
Oops, sorry I slipped on this. People without write access need to have their PRs merged for them, right? It's been a while.

**Comment by emdupre** [comment]
> People without write access need to have their PRs merged for them, right? 

Yes :smile_cat: If you also approve, I'll let you pull the trigger !

**Comment by tsalo** [comment]
@all-contributors please add @aykhojandi for documentation

**Pull request merged**